### ISSUE TITLE
I hope this repo is not dead and somehow I get answers - own shortcuts.

### ISSUE THREAD

**Issue by MattOrmianek** [mjrusso/scoot#49]
I understand that it's possible to change shortcuts for modes, but it would be great to have the ability to edit other shortcuts as well. I can't find this option anywhere, and it seems like a fundamental feature. Even a simple JSON file in the .config directory would suffice.

**Comment by Reefersleep** [comment]
I agree, especially because I can't get `[`, `\` and `]` to do anything on my computer, so it'd be great to be able to remap to something that worked :)
